packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -73,7 +73,7 @@ function useAddAppMutation(_type: App["type"] | null, options?: UseAddAppMutatio
       gotoUrl(json.url, json.newTab);
     }
 
-    return { setupPending: !externalUrl && json.url.endsWith("/setup") };
+    return { setupPending: externalUrl || json.url.endsWith("/setup") };
```

Comment:
This somehow was wrong, corrected it. We want to let the UI know there is setup pending when a redirection is going to be done to an external URI or the URL to redirect URL ends with "/setup", which means there is an internal setup page pending to be filled in order to consider the app successfully installed.